index.html

```diff
@@ -567,6 +646,22 @@ <h1>Exploration Mechanisms</h1>
                         information referencing another <a>TD</a> which contains the full description.
                     </li>
                 </ul>
+		The `hosts` and `external` relations are meant to support the following use case:
+		<ul>
+		<li>A single device, IP address, or server hosts multiple Things.  However, some Introduction mechanisms
+		only permit one URL per device, and each URL can only return one TD.  In order to supply
+		multiple TDs from a single URL without introducing a TDD, a Think Link can be returned
+		as the permitted single TD, but can then reference multiple TDs.
+		</li>
+	        </ul>
+		Note that the `hosts` and `external` relation type, and Thing Links in general, do not
+		necessarily describe collections of TDs.  Things may be co-hosted under a common base URL
+		and/or physical device (with a single IP address, for example) 
+		without being directly related to each other.  
+		If a set of Things <em>are</em> part of a collection, additional Links should be provided
+		in the individual TDs to describe such relations, for example using `collection` and
+		`item` relations.  
```

Comment:
I think this paragraph will add a lot of confusion because it directly overlaps with the functionalities of directories and other relation types defined above it. In this spec, we have numerous references to "collection", a JSON-LD class called "ThingCollection" and listing format "collection"; all referring to what is stored in the directory or the list of TDs.

Given that, can a Link with "collection" rel point to a directory? Because TD [spec](https://w3c.github.io/wot-thing-description/#link) says: "collection: Points to a collections of Things". What will be the difference from a TD with type ThingDirectory and "describedby" link referring to that directory?

Looking at the RFC that defined that relation:
>   When included in a resource that represents a member of a collection,
   the 'collection' link relation identifies a target resource that
   represents a collection of which the context resource is a member.
https://www.rfc-editor.org/rfc/rfc6573.html#section-2.2

If I understand correctly, this means that the TD pointing to that link must be part of the linked "collection". 

> When included in a resource that represents a collection, the 'item'
   link relation identifies a target resource that represents a member
   of that collection.
https://www.rfc-editor.org/rfc/rfc6573.html#section-2.1

Is the linked "item" a member of that parent TD "collection"? Is the parent TD even a collection?

Reply:
ok, I'll think about it.  I could just take this out but I do want to emphasize that is this just an example.  It's a lot easier to do the iterative discovery process if there is no important information stored in Thing Links, so they can be elided as part of the process.  BTW I just expanded the arch intro to describe that process as well (including updating the diagram).